example/src/examples/TiledBuilding.tsx

```diff
@@ -0,0 +1,209 @@
+import React, { useEffect, useState } from "react";
+import { View, ActivityIndicator, Image, Dimensions } from "react-native";
+
+import SitumPlugin from "react-native-situm-plugin";
+import MapView, { PROVIDER_GOOGLE, Overlay, Marker , UrlTile, MapLocalTile} from "react-native-maps";
+import { SITUM_BUILDING_ID } from "../situm";
+
+
+
+
+//This example shows how to display a tiled floorplan hosted in Situm Platform.
+//Floorplans are not stored in tiles by default, but Situm Support Team can tile your floorplans & upload them to Situm Platform (ask us: support@situm.com)
+//Change TILES_URL to the URL provided by Situm Support.
+ 
+
+const MAX_ZOOM_LEVEL = 20
+
+
+// This URL will be provided by Situm Support. {building_id} and {floor_id} will be the ID of the building and that you want to display, respectivelly.
+// {z},{y}, {x} are the tile selection variables that will be auto-filled at runtime by react-native-maps
+const TILES_URL = "https://dashboard.situm.com/uploads/{building_id}/{floor_id}/{z}/{x}/{y}.png" 
+
+
+export const TiledBuilding = () => {
+  const [building, setBuilding] = useState<any>();
+  const [buildings, setBuildings] = useState<any>();
+  const [selectedBuilding, setSelectedBuilding] = useState<any>();
+  const [mapRegion, setMapRegion] = useState<any>();
+  
+  const [isLoading, setIsLoading] = useState<Boolean>(false); 
+  const [error, setError] = useState<String>();
+  const [floor, setFloor] = useState<any>();
+
+  const [hasStoredTiles, setHasStoredTiles] = useState<Boolean>(false); 
+  const [offlineTilePath, setOfflineTilePath] = useState<String>();
+  
+
+
```

Comment:
Como comentario xeral para este ficheiro, podes quitar o código comentado que non sexa neceario? Así queda máis limpo o exemplo.

Hai tamen varias liñas en branco que penso que se poden eliminar